src/site/content/en/blog/file-system-access/index.md

```diff
@@ -234,6 +234,75 @@ file at a specific position, or resize the file.
   calling `close()` or when the stream is automatically closed by the pipe.
 {% endAside %}
 
+### Specifying a suggested file name and start directory
+
+In many cases your application may already have an opinion when it comes to default file names.
+For example, a text editor might want to suggest a default file name of `Untitled Text.txt`
+rather than `Untitled`. You can achieve this by passing a `suggestedName` property as part
+of the `showSaveFilePicker` options.
+
+```js/1
+const fileHandle = await self.showSaveFilePicker({
+  suggestedName: 'Untitled Text.txt',
+  types: [{
+    description: 'Text documents',
+    accept: {
+      'text/plain': ['.txt'],
+    },
+  }],
+});
+```
+
+The same goes for the default start directory. A text editor may want to start the file save
+or file open dialog in the default `documents` folder, whereas an image editor may want to
+start in the default `pictures` folder. You can suggest a default start directory by passing
+a `startIn` property to the `showSaveFilePicker`, `showDirectoryPicker()`, or `showOpenFilePicker`
+methods like so.
+
+```js/1
+const fileHandle = await self.showOpenFilePicker({
+  startIn: 'pictures'
+});
+```
+
+The list of the well-known system directories is:
+
+- `desktop`: The user's desktop directory, if such a thing exists.
+- `documents`: Directory in which documents created by the user would typically be stored.
+- `downloads`: Directory where downloaded files would typically be stored.
+- `music`: Directory where audio files would typically be stored.
+- `pictures`: Directory where photos and other still images would typically be stored.
+- `videos`: Directory where videos/movies would typically be stored.
```

Comment:
I haven't used Windows in years. How platform independent is this list?

Reply:
Independent enough to work on CrOS, macOS, Windows, and Linux (I think the distribution was Ubuntu).